lib/Page.js

```diff
@@ -134,10 +131,13 @@ Page.prototype.resolveDependency = function (dependency, builtFiles) {
   const source = dependency.from;
   const file = dependency.to;
   return new Promise((resolve, reject) => {
-    const markbinder = new MarkBind();
     const resultDir = path.dirname(path.resolve(this.resultPath, path.relative(this.sourcePath, file)));
     const resultPath = path.join(resultDir, FsUtil.setExtension(path.basename(file), '._include_.html'));
 
+    const markbinder = new MarkBind({
+      errorHandler: logger.error,
+    });
```

Comment:
`markbinder` can be instantiated after the potential `return` (and `builtFiles[resultPath] = true;`).

As mentioned, let's add your explanation as a comment here.